solidity_deposit_contract/README.md

```diff
@@ -23,3 +23,16 @@ Install the latest version of `dapp` by following the instructions at [dapp.tool
 ```sh
 make test_deposit_contract
 ```
+## Verifying bytecode
```

Comment:
Suggest moving this section to before "Running web3 tests" section and adding:

```suggestion
## Compiling solidity

In the `eth2.0-specs` directory run:
```sh
make compile_deposit_contract
```

```